src/TestProblem/ELBDM/LSS/Init_TestProb_ELBDM_LSS.cpp

```diff
@@ -4,7 +4,14 @@
 
 // problem-specific global variables
 // =======================================================================================
-static int LSS_InitMode;   // initialization mode: 1=density-only, 2=real and imaginary parts or phase and density
+static int  LSS_InitMode;   // initialization mode: 1=density-only, 2=real and imaginary parts or phase and density
+static double ZoomIn_Center_x;  // x coordinate of refinement region center
+static double ZoomIn_Center_y;  // y coordinate of refinement region center
+static double ZoomIn_Center_z;  // z coordinate of refinement region center
+static int ZoomIn_Lvlim;        // maximum refinement level outside of the zoom-in box
+static int Max_snap = 30;       // number of changes the zoom-in box. could be changed here if a higher freqeuncy is required.
```

Comment:
size of array must be an compile time constant, so it is not possible to read ` Max_snap`  at run-time and creates array with that size.

Reply:
Done. 
In addition,  since its better to have `Max_snap `to be same as number of rows in `Input__ZoominBox`.
`Max_snap `is now not required to be specified in` Input__TestProb`.